com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -1592,10 +1592,15 @@ private void transpileLua(IProgress monitor) throws CompileExceptionError, IOExc
                 IResource buildFileResource = getResource(transpiler.getBuildFileResourcePath());
                 if (buildFileResource.exists()) {
                     String ext = "." + transpiler.getSourceExt();
-                    List<IResource> sources = inputs.stream()
-                            .filter(s -> s.endsWith(ext))
-                            .map(this::getResource)
-                            .collect(Collectors.toList());
+                    ArrayList<IResource> sources = new ArrayList<>();
+                    fileSystem.walk("", new FileSystemWalker() {
+                        @Override
+                        public void handleFile(String path, Collection<String> results) {
+                            if (path.endsWith(ext)) {
+                                sources.add(fileSystem.get(path));
+                            }
+                        }
+                    }, new ArrayList<>());
```

Comment:
I don't like us having this extra step outside our system with Builders, but if it's already implemented this way and works, let's keep it